css/DesktopVersion.css

```diff
@@ -0,0 +1,365 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: #dfe1e6;
+  }
+
+  header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    background-color: white;
+    color: black;
+    padding: 10px;
+  }
```

Comment:
- Kindly make Sure replicate the design as much as possible (e.g., font, colors, images, text, margins) using [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please pay Close attention to the margin in the navbar, the text color, the background image, the link color and the icons position.

| Figma Design  | Your Design |
| ------------- | ------------- |
|<img src="https://user-images.githubusercontent.com/97834160/236323728-d27a17c4-f2ab-47f2-8c78-10506f3b6eb8.png" width="500"> | <img src="https://user-images.githubusercontent.com/97834160/236322785-ff2d7cb2-216f-4683-b3aa-b818d9ece9c1.png" width="500">|

- Kindly make Sure replicate the design as much as possible (e.g., font, colors, images, text, margins) using [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

| Figma Design  | Your Design |
| ------------- | ------------- |
|<img src="https://user-images.githubusercontent.com/97834160/236324382-16a69e9d-f8c7-4de9-aa6b-c26cfc703430.png" width="500"> | <img src="https://user-images.githubusercontent.com/97834160/236324613-3b4bcc4b-6292-4159-8b94-b2b7ea83c738.png" width="500">|

- Kindly make Sure replicate the design as much as possible (e.g., font, colors, images, text, margins) using [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

| Figma Design  | Your Design |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/97834160/236324952-da3b156a-5622-4126-9c35-958ed65be842.png" width="500">|<img src="https://user-images.githubusercontent.com/97834160/236324974-1d3087fd-5d25-4ad2-b2f5-b1790563064e.png" width="500"> |

- Kindly make Sure replicate the design as much as possible (e.g., font, colors, images, text, margins) using [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

| Figma Design  | Your Design |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/97834160/236325437-68b7de2d-3aa7-47aa-81c0-9ef5a3859f0e.png" width="500">|<img src="https://user-images.githubusercontent.com/97834160/236325465-a417afd5-076d-42a0-8837-4e8fe12abdc6.png" width="500"> |

- Kindly make Sure replicate the design as much as possible (e.g., font, colors, images, text, margins) using [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).  Please use the desktop background provided by the Figma template and pay close attention to the spaces, margins,...

| Figma Design  | Your Design |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/97834160/236325708-bc6718e7-8517-44b8-bcf6-4644a0db8313.png" width="500">|<img src="https://user-images.githubusercontent.com/97834160/236325736-c1319b91-c9fa-4404-b094-f6dbf43f19ac.png" width="500"> |

**NOTE: As the number of Code Review requests is limited, I'd recommend you to do your best to stick to all the requirements and make finish everything for your desktop version before submitting another review request**